src/nifgen/metadata/enums_addon.py

```diff
@@ -8,4 +8,18 @@
 enums_codegen_method = {
 }
 
+enums_additional_enums = {
+    'RelativeTo': {
+        'values': [
+            {
+                'name': 'START',
+                'value': 0,
```

Comment:
Do we need prefix key as 'WAVEFORM_POSITION' ?

Reply:
@marcoskirsch and I discussed this. Instead of removing the common prefix, we think you should make the name match the C API completely. Then the codegen step can operate like normal and it will put the prefix in the processed metadata for you. So this 'name' should be changed to `NIFGEN_VAL_WAVEFORM_POSITION_START` and the other should be changed to `NIFGEN_VAL_WAVEFORM_POSITION_CURRENT`.

Sorry about changing this again.